charts/pgadmin4/templates/deployment.yaml

```diff
@@ -71,8 +75,17 @@ spec:
               mountPath: /pgadmin4/servers.json
               subPath: servers.json
           {{- end }}
+          {{- range .Values.extraConfigmapMounts }}
+            - name: {{ .name }}
+              mountPath: {{ .mountPath }}
+              subPath: {{ .subPath | default "" }}
+              readOnly: {{ .readOnly }}
+          {{- end }}
           resources:
             {{- .Values.resources | toYaml | nindent 12 }}
+{{- with .Values.extraContainers }}
+{{ tpl . $ | indent 8 }}
```

Comment:
`nindent` is almost identical to indent, but begins a new line. Makes the file much more readable. Could you perhaps use that here?